runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/LargeCommitTest.java

```diff
@@ -19,14 +19,18 @@
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 
+import com.google.common.collect.Iterables;
```

Comment:
nit: spotless is complaining about this 

```suggestion
import org.apache.beam.vendor.guava.v32_1_2_jre.com.google.common.collect.Iterables;
```